<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeworkGUIClass</class>
 <widget class="QMainWindow" name="HomeworkGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HomeworkGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="tuan1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Tuần 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tuan2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Tuần 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tuan3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Tuần 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lựa chọn tuần</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="HomeworkGUI.qrc"/>
 </resources>
 <connections/>
</ui>
